WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Carriers and Rates/ShippingLabelCarriersAndRates.swift

```diff
@@ -0,0 +1,60 @@
+import SwiftUI
+import Yosemite
+
+struct ShippingLabelCarriersAndRates: View {
```

Comment:
It seems that Apple changed the way we should use the `*View` suffix. In fact, in SwiftUI all the components don't have the `*View` suffix: it's `VStack` not a `VStackView`. We used some `*View` suffix in the past for some SwiftUI views in our app, but probably we should update them.
I renamed it `ShippingLabelCarriers` because the rates are part of the carriers. 👍

Reply:
I see! Probably something I have to get used to then.